src/TimeModeling/judiWavefield.jl

```diff
@@ -256,6 +256,21 @@ function isapprox(x::judiWavefield, y::judiWavefield; rtol::Real=sqrt(eps()), at
     isapprox(x.data, y.data; rtol=rtol, atol=atol)
 end
 
+
+function diff(a::judiWavefield{avDT}; dims=1) where avDT
+	b = deepcopy(a)
+	for j=1:a.info.nsrc
+		s = 1
+		@views loc = b.data[j]
+		for d in dims
+			s *= dims == 1 ? 1/a.dt : 1
+			loc = selectdim(loc, d, 2:size(loc, d))
+		end
+		loc .= s*diff(a.data[j], dims=dims)
+	end
+	return b
+end
```

Comment:
Should we also implement a cumsum for judiWavefield? Also, could you add a test for it?

Reply:
If is needed should have been added in the cumsum/diff PR, so I'm just gonna remove it.

Reply:
Sure. I think everything is in good shape now.